SPECIFICATION.md

```diff
@@ -0,0 +1,368 @@
+  "versions": [],
+  "published": "2023-06-01T00:00:00Z",
+  "modified": "2023-06-01T00:00:00Z",
+}
+```
+
+#### endOfLife
+*Category: Version Event*
+
+Indicates when the manufacturer stops distribution of a product after its defined useful life and formally notifies users. It marks the conclusion of the product's lifecycle, following a structured EOL process.
+
+**Additional Required Fields:**
+
+| Field | Type | Description |
+|-------|------|-------------|
+| `versions` | array[string] | Array of simple version strings (e.g., ["1.0.0", "1.1.0"]). Required for version-specific events. |
+| `ranges` | array[string] | Array of VERS-formatted version ranges (e.g., ["vers:npm/>=1.0.0|<2.0.0"]). Required for range-based events. |
+
+Example:
+```json
+{
+  "type": "endOfLife",
+  "effective": "2024-01-01T00:00:00Z",
+  "ranges": ["vers:npm/>=1.0.0|<2.0.0"],
+  "versions": [],
+  "published": "2023-06-01T00:00:00Z",
+  "modified": "2023-06-01T00:00:00Z",
+}
+```
+
+#### endOfProduction
+*Category: Version Event*
+
+Indicates when the manufacturer stops producing a component, often due to newer versions, unavailable parts, market changes, or strategic shifts. Existing units may still be available in warehouses, distribution channels, and for use, but no new units will be manufactured.
+
+**Additional Required Fields:**
+
+| Field | Type | Description |
+|-------|------|-------------|
+| `versions` | array[string] | Array of simple version strings (e.g., ["1.0.0", "1.1.0"]). Required for version-specific events. |
+| `ranges` | array[string] | Array of VERS-formatted version ranges (e.g., ["vers:npm/>=1.0.0|<2.0.0"]). Required for range-based events. |
+
+Example:
+```json
+{
+  "type": "endOfProduction",
+  "effective": "2024-01-01T00:00:00Z",
+  "ranges": ["vers:npm/>=1.0.0|<2.0.0"],
+  "versions": [],
+  "published": "2023-06-01T00:00:00Z",
+  "modified": "2023-06-01T00:00:00Z",
+}
+```
+
+#### endOfMarketing
+*Category: Version Event*
+
+Indicates when the manufacturer will cease actively promoting or advertising a component or service. While the component or service may still be available for purchase and support, it will no longer receive active marketing efforts from the manufacturer.
+
+**Additional Required Fields:**
+
+| Field | Type | Description |
+|-------|------|-------------|
+| `versions` | array[string] | Array of simple version strings (e.g., ["1.0.0", "1.1.0"]). Required for version-specific events. |
+| `ranges` | array[string] | Array of VERS-formatted version ranges (e.g., ["vers:npm/>=1.0.0|<2.0.0"]). Required for range-based events. |
+
+Example:
+```json
+{
+  "type": "endOfMarketing",
+  "effective": "2024-01-01T00:00:00Z",
+  "ranges": ["vers:npm/>=1.0.0|<2.0.0"],
+  "versions": [],
+  "published": "2023-06-01T00:00:00Z",
+  "modified": "2023-06-01T00:00:00Z",
+}
+```
+
+#### supersededBy
+*Category: Version Event*
+
+Indicates when a version of a component is superseded by another version of a component.
```

Comment:
isn't every version of a component superseded by all future versions? what's an example where this event isn't always implied?